sklearn/tests/test_multiclass.py

```diff
@@ -824,6 +824,49 @@ def test_pairwise_indices():
         )
 
 
+def test_pairwise_n_features_in():
```

Comment:
Maybe a docstring (or comment) explaining high level what this test does.

Reply:
Done below, let me know what you think: